README.adoc

```diff
@@ -185,11 +209,23 @@ Greetings from Spring Boot!
 ----
 ====
 
-== Add Unit Tests
+== Add Tests
 
 You will want to add a test for the endpoint you added, and Spring Test provides some
 machinery for that.
 
+You should see the following dependency in your `build.gradle` file:
+```
+testImplementation 'org.springframework.boot:spring-boot-starter-test'
+```
+
+NOTE: This is one of http://docs.spring.io/spring-boot/docs/3.1.1/reference/htmlsingle/#using-boot-starter[Spring Boot’s
+"`starters`"]. A starter contains pre-configured bundles of dependencies which allow you to get going quickly without configuring your own dependencies.
+We'll use additional starters in this guide. You can see the complete collection of Spring Boot - supplied starters https://github.com/spring-projects/spring-boot/tree/main/spring-boot-project/spring-boot-starters[in the Starter repository at GitHub].
```

Comment:
This might be a better link https://docs.spring.io/spring-boot/docs/current/reference/html/using.html#using.build-systems.starters

Reply:
We definitely want to link to current and to the more specific location in a guide that spans several hundred pages (when printed as PDF).